java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -62,4 +87,4 @@ public static void main(String[] args) {
         }
         log.info("Received {} messages", receivedMsgs);
     }
-}
+}
```

Comment:
I think we can keep the newline

Reply:
Do you mean an extra empty line at the bottom? If so, added.